corehq/motech/fhir/views.py

```diff
@@ -0,0 +1,29 @@
+from django.http import JsonResponse
+
+from corehq import toggles
+from corehq.apps.domain.decorators import require_superuser, login_and_domain_required
+from corehq.form_processor.exceptions import CaseNotFound
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from corehq.motech.fhir.models import FHIRResourceType, build_fhir_resource
+
+
+@login_and_domain_required
+@require_superuser
+@toggles.FHIR_INTEGRATION.required_decorator()
+def get_view(request, domain, resource_type, resource_id):
+    try:
+        case = CaseAccessors(domain).get_case(resource_id)
+        if case.is_deleted:
+            return JsonResponse(status=400, data={'message': f"Resource with ID {resource_id} was removed"})
+    except CaseNotFound:
+        return JsonResponse(status=400, data={'message': f"Could not find resource with ID {resource_id}"})
+
+    if case.type not in (FHIRResourceType.objects.filter(domain=domain, name=resource_type).
+                         values_list('case_type__name', flat=True)):
+        return JsonResponse(status=400, data={'message': "Invalid Resource Type"})
+    response = {
+        'resourceType': resource_type,
+        'id': resource_id
+    }
+    response.update(build_fhir_resource(case))
```

Comment:
This should end up as
```suggestion
    response = get_fhir_resource_for_case(case, fhir_version)
```

Reply:
are those added by `build_fhir_resource` now?

Reply:
Added in [this commit](https://github.com/dimagi/commcare-hq/pull/29261/commits/b0ee0d495f3b1687f64b96343427963e5c6f8bf4#diff-433e7032abf02a0c78231568078bff8181998607bf011edb22ea4b1d3018041fR199-R200), in PR #29261

Reply:
Cool, that PR is still in draft state, and this one is in QA. So going to keep this as is for now and make a note to update this later.